### DISCUSSION TITLE
[@types/chrome] Specifies argument as `number`, but it's actually `number|null`.

### DISCUSSION THREAD

**Discussion by danon** [DefinitelyTyped/DefinitelyTyped#70723] [Issues with a @types package]
When calling `chrome.tabs.executeScript()`, the first argument can be `null`, and it indicates "the current tab". But the type says it must be `number`.

[a comment by an automated account is not shown]

**Comment by erwanjugand** [comment]
Hello @danon, looking at [the documentation](https://developer.chrome.com/docs/extensions/mv2/reference/tabs#method-executeScript), `tabId` is a number and optional.

You can do : `chrome.tabs.executeScript(details)`

**Reply by danon** [reply]
Yes, it is optional, and yet the current type specifies it's not optional.

**Reply by erwanjugand** [reply]
However there is a type explaining that `tabId` is optional : https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/chrome/index.d.ts#L10495

Currently you can write : 
```ts
const details = {}
const tabId = 0;

await chrome.tabs.executeScript(details);
chrome.tabs.executeScript(details, () => {});

await chrome.tabs.executeScript(tabId, details);
chrome.tabs.executeScript(tabId, details, () => {});
```

Could you share a minimal reproduction? :)
